pkgs/by-name/al/alphafill/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost,
+  date,
+  libmcfp,
+  libcifpp,
+  libzeep,
+  mrc,
+  zlib,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "alphafill";
+  version = "2.1.0";
+
+  src = fetchFromGitHub {
+    owner = "PDB-REDO";
+    repo = "alphafill";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-vXzylvecsYDxmQ2r9W8aQsMYZqzY1mIIQLJHUCDWi9k=";
+  };
+
+  prePatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-fail "1.2.4" "" \
+      --replace-fail "6.0.11" "" \
+      --replace-fail "QUIET" ""
```

Comment:
I feel that for whatever you're trying to do here, `sed` and a regex might be more appropriate.

Reply:
May you proposed an expression to replace `find_package\((?<name>\w+)\s+(?=\d|[.]+)\s+(?=QUIET)?\)` by `find_package(<name>)`? I have no clue how to do this using `sed`